parser.js

```diff
@@ -34,7 +34,7 @@ function parseTweet(tweet) {
 }
 
 function getTweets(threadReaderDoc) {
-  return queryDOM(threadReaderDoc)('.t-main .content-tweet').map(parseTweet);
+  return queryDOM(threadReaderDoc)('.content-tweet').map(parseTweet);
```

Comment:
I tried this out, and it pulled content from the start of other threads recommended down the bottom. I filtered those out by using
```suggestion
  return queryDOM(threadReaderDoc)('.content-tweet.allow-preview').map(parseTweet);
```